lib/jquery-rjs/rendering.rb

```diff
@@ -1,13 +1,21 @@
 require 'action_view/helpers/rendering_helper'
 
-ActionView::Helpers::RenderingHelper.module_eval do
-  def render_with_update(options = {}, locals = {}, &block)
-    if options == :update
-      update_page(&block)
-    else
-      render_without_update(options, locals, &block)
-    end
+module JqueryRjs::RenderingHelper
+  method_name = "render#{'_with_update' if RUBY_VERSION < '2'}"
```

Comment:
First code block: No, method must be named `render` to be properly inserted in the inheritance tree with `prepend` (must be able to call `super`)

> Something to keep in mind is you are also changing the functionality of the function call...

No, not intended, good catch. That's a typo (a bad one).